README.md

```diff
@@ -34,15 +35,16 @@ class MyModelBlock(nn.Module):
     y = self.layer_norm(x)
     y = self.linear_hidden(y)
     y = nn.sigmoid(y)
-    y = self.linear_out(x)
-    y = nn.dropout(y, dropout=self.dropout)
+    y = self.linear_out(y)
+    y = nn.dropout(y, dropout=self.dropout, axis=[nn.batch_dim])
```

Comment:
This doesn't really make sense. You don't want to dropout whole sequences. You usually would want this:
```suggestion
    y = nn.dropout(y, dropout=self.dropout, axis=nn.any_feature_dim)
```